### DISCUSSION TITLE
Stable release of ErgoAI 3.0

### DISCUSSION THREAD

**Discussion by kifer** [ErgoAI/.github#5] [Announcements]
This is the first stable release of the open-source version of [ErgoAI](https://github.com/ErgoAI).
ErgoAI is an extension and a replacement for the popular [Flora-2](https://flora.sourceforge.net/) KRR system.

To install this release, download the file intended for your system into a suitable folder.
For Linux/Mac, this would be the directory where you are planning to install ErgoAI. For Windows, the installation directory can be chosen during the installation process.  

*  Linux/Mac:  [ergoAI_3.0.run](https://github.com/ErgoAI/.github/releases/download/v3.0_release/ergoAI_3.0.run)
    * Open a terminal window
    * `cd` to the download directory
    * type `sh ergoAI_3.0.run` in the terminal window and answer the prompts
    * at the end, two icons for running ErgoAI should appear on your desktop
* Windows:  [ergoAI_3.0.exe](https://github.com/ErgoAI/.github/releases/download/v3.0_release/ergoAI_3.0.exe)
   * Place the downloaded file on your desktop
   * double-click on the resulting file icon and click through the installation process
   * at the end, two icons for running ErgoAI should appear on your desktop

More details can be found in [About ErgoAI](https://github.com/ErgoAI).

<hr /><em>This discussion was created from the release <a href='https://github.com/ErgoAI/.github/releases/tag/v3.0_release'>Stable release of ErgoAI 3.0</a>.</em>

**Comment by yagupta** [comment]
**ErgoAI Studio 3.0 does not launch**

I am using a MacBook Air M2 (Apple Silicon) w/ Ventura 13.4.1(c). I am able to install ErgoAI, the Reasoner works fine but the Studio doesn't launch.

**Comment by thomas-e314** [comment]
ErgoAI Studio 3.0 does not launch

I installed ErgoAI Studio using the Windows installer (see link above ). ErgoAI Reasoner starts, ErgoAI Studio claims not to find the Prolog engine.
